docs/docs/integrations/providers/databricks.md

```diff
@@ -28,6 +28,12 @@ Databricks MLflow makes it more convenient to develop LangChain applications on
 Databricks MLflow AI Gateway
```

Comment:
@harupy Can we also update this page to show simpler completions examples for external / foundation model endpoints, and also show examples for chat & embeddings?

Reply:
`docs/docs/integrations/llms/databricks.ipynb` contains examples for external / foundation model endpoints. Can we use them?

Reply:
This section is out of date: https://github.com/langchain-ai/langchain/blob/fc02bb88b1633cbd9f7290fd33aeef1091f29f63/docs/docs/integrations/providers/databricks.md#databricks-as-an-llm-provider

Reply:
Got it, I'll update them.